cmake/test.cmake

```diff
@@ -178,6 +180,14 @@ finally:
          "${CMAKE_INSTALL_PREFIX}/plugin/al/lib/usd")
     list(APPEND mayaUsd_varname_PXR_PLUGINPATH_NAME
          "${CMAKE_INSTALL_PREFIX}/plugin/al/plugin")
+    if (IS_LINUX AND CMAKE_SKIP_RPATH)
```

Comment:
Same here - part of #464 - please remove.

Reply:
You're asking me to submit a PR with it being unable to pass our internal integration tests. At this point, I'd rather submit a PR that removes mayaUsd/ufe from maya-usd so that I can actually do some fucking work.